pkgs/development/python-modules/inquirer/default.nix

```diff
@@ -20,7 +20,7 @@
 
 buildPythonPackage rec {
   pname = "inquirer";
-  version = "3.2.5";
+  version = "3.3.0";
   format = "pyproject";
```

Comment:
```suggestion
  pyproject = true;
```

Reply:
We should probably avoid these changes to r-ryantm's PR since someone else will have to take ownership of the PR.

Reply:
You could open a new PR.

Reply:
@mmahut (package maintainer) may be better suited for it.